robottelo/ui/repository.py

```diff
@@ -169,6 +169,8 @@ def remove_content(self, repo_name):
         """Remove content from a repository."""
         self.search_and_click(repo_name)
         self.click(locators['repo.manage_content'])
+        # fixme: Should be replaced with conditional loop for >100 packages
+        self.assign_value(common_locators['table_per_page'], '100')
```

Comment:
Basically, it will work will our fake repos (which have 32 packages), but i guess it's better to use some complex solution with conditional loop. Submitted #4829 not to forget about it.